src/Domain/Creators/SphereTimeDependentMaps.cpp

```diff
@@ -115,17 +133,21 @@ void TimeDependentMapOptions::build_maps(const std::array<double, 3>& center,
   shape_map_ = ShapeMap{center,         initial_l_max_,
                         initial_l_max_, std::move(transition_func),
                         shape_name,     size_name};
+  if (translation_inner_radius == translation_outer_radius) {
```

Comment:
Take the translation radii as `std::optional<std::pair<double, double>>` then you can either pass `std::nullopt` or both radii.

You also have to build a second rigid translation map with no radii, which you can apply to the inner blocks. Then you don't need to remove asserts in the Translation map. I think it is necessary to have these two different maps because their Jacobian is discontinuous at the `translation_inner_radius`.